src/osuTK.Android/GameViewBase.cs

```diff
@@ -1139,6 +1139,8 @@ Icon INativeWindow.Icon
             set { throw new NotSupportedException(); }
         }
 
+        public bool CursorGrabbed { get => throw new NotImplementedException(); set => throw new NotImplementedException(); }
```

Comment:
To be honest I would force this to always be true. We can’t assume that anything cursor-related is unsupported, since there are mice for tablets.

Reply:
Then you must guarantee that it always grabbed

Reply:
Regardless, I would rather a default functionality than an exception.  We can do a full implementation later, but I don't want to be explicitly avoiding calls to things that will throw on mobile.